recipe-client-addon/test/browser/browser_NormandyDriver.js

```diff
@@ -12,6 +12,16 @@ add_task(withDriver(Assert, async function uuids(driver) {
   isnot(uuid1, uuid2, "uuids are unique");
 }));
 
+add_task(withDriver(Assert, async function installXpi(driver) {
+  // Test that we can install an XPI from any URL
+  const dir = getChromeDir(getResolvedURI(gTestPath));
+  dir.append("fixtures");
+  dir.append("normandy.xpi");
+  const xpiUrl = Services.io.newFileURI(dir).spec;
+  var addonId = await driver.installAddon(xpiUrl);
+  await driver.uninstallAddon(addonId);
```

Comment:
If this ever fails (partially or fully), I expect checking explicitly that `addonId` is not undefined (or whatever else `installAddon` returns if installation didn't work) and/or that there's an installed add-on with the returned ID, will result in more understandable failure messaging than "uninstallAddon threw an error" (because the install failed).

More generally, I'm surprised the ID would be what this returns, and that's all that it returns - the ID is available without installing the add-on...

Also, late nit: `let` please. :-)